docs/source/API/core/view/view.rst

```diff
@@ -0,0 +1,577 @@
+``View``
+========
+
+.. role:: cppkokkos(code)
+    :language: cppkokkos
+
+Header File: ``<Kokkos_Core.hpp>``
+
+Usage
+-----
+
+Kokkos View is a potentially reference counted multi dimensional array with compile time layouts and memory space.
+Its semantics are similar to that of ``std::shared_ptr``.
+
+Interface
+---------
+
+.. code-block:: cpp
+
+    template <class DataType [, class LayoutType] [, class MemorySpace] [, class MemoryTraits]>
+    class View;
+
+Parameters
+~~~~~~~~~~
+
+.. _LayoutRight: layoutRight.html
```

Comment:
Just a note that we will have to replace these with cross-references once everything is ported over

Reply:
yes